eclass/kernel-build.eclass

```diff
@@ -213,6 +206,32 @@ kernel-build_src_install() {
 	dosym ../../../usr/src/linux-${ver} /lib/modules/${ver}/build
 	dosym ../../../usr/src/linux-${ver} /lib/modules/${ver}/source
 
+	# Install the unstripped uncompressed vmlinux for use with systemtap
+	# etc.  Use mv rather than doins for the same reason as above --
+	# space and time.
+	if use debug; then
+		local vmlinux_path="build/vmlinux"
+		local debug_path="/usr/lib/debug"
+		debug_path="${debug_path}/lib/modules/${ver}"
```

Comment:
Why do you need to do it like this?

Reply:
Just wanted to avoid the long line, but now when I look at it it's probably even worse.